site/content/api/Event-API.md

```diff
@@ -76,4 +81,63 @@ The following sequence diagram illustrates the flow of messages involved in a *B
 
 **Message Format**
 
+**Arbitrary events from the device**
+
 See [*Telemetry API*]({{< relref "Telemetry-API.md" >}}) for definition of message format. 
+
+## Well-known event message types
+
+Hono defines *well-known* events that are of a specific *content-type*.
+
+In the following these events (currently one) are specified in detail.
+
+
+### Device command readiness notification
+
+By sending an event of this type the receiver is notified that a device is ready to receive a command.
+ 
+*Business Applications* may want to react to such an event by sending a command upstream to the device.
+For that purpose, the protocol adapter instance that received such an event opens a receiver link for the device (scoped to its tenant) that
+is used to send the command to the appropriate protocol adapter instance.
+
+Such an event is typically issued by the device itself. In future releases the protocol adapters may be able to send this
+event on behalf of devices (to support setups with devices that are not capable of sending such an event).
+
+
+***Device command readiness notification Payload***
+
+The following table provides an overview of the properties an event sender needs to set for a *device command readiness notification* event.
+
+| Name           | Mandatory | Location                 | Type      | Description |
+| :------------- | :-------: | :----------------------- | :-------- | :---------- |
+| *content-type* | yes       | *properties*             | *symbol*  | Must be set to *application/vnd.eclipse-hono-dcr-notification+json* |
```

Comment:
what about `application/vnd.eclipse-hono-ready-to-receive+json`

Reply:
Since it is used in a lot of places already, I would rather prefer
`application/vnd.eclipse-hono-ready-for-upstream+json`.

I would be uncertain as a reader about `who` is ready to receive otherwise...

WDYT?

Reply:
It defines a semantic (about what should happen) with this content-types and I am not sure if this is not to much. What is indicated is a stay-connected-timeframe. Maybe we should only indicate that and do not care, what the receiver will do with this information.

Reply:
I don't like that name either. If we are able to get rid of the payload, we can also use `plain/empty` ;-)

Reply:
>  What is indicated is a stay-connected-timeframe

I thought differently: indicated is "ready to receive a message".
"Stay connected" is not the same for me as "willing to receive a message".

Since we have no payload anymore, we need a new type now, probably something like
`application/vnd.eclipse-hono-...+empty`

Any ideas?

Reply:
`application/vnd.eclipse-hono-empty-notification` ?

Reply:
Taken :+1: